### PR TITLE
fix: Fix example to match Marshmallow TZ semantics

### PR DESCRIPTION
**Pull request by taion**

### CODE REVIEW

File: docs/guide.rst [hunk 1]
```diff
@@ -298,16 +298,16 @@ You can make requests using the `httpie <https://httpie.org/>`_ utility.
         "data": [
             {
                 "author_id": 2,
-                "created_at": "2019-06-16T01:09:33.450768+00:00",
+                "created_at": "2019-06-16T01:09:33.450768",
```
Comment (taion) [inline docs/guide.rst]: Ick.

Maybe I should add a note that these are only naive because it's SQLite on the back end here?

Comment (taion) [reply docs/guide.rst]: did that

**Comment by taion** [comment]
@sloria boo i think you need to cherry-pick this to your branch actually. how does this look though?

**Comment by taion** [comment]
@sloria can you cherry-pick this into #293, merge that, then close this?

**Comment by sloria** [comment]
Yes will do

**Pull request closed without merging**